src/shared/frame-rpc.js

```diff
@@ -117,45 +148,105 @@ export class RPC {
     if (this._destroyed) {
       return;
     }
+
     const seq = this._sequence++;
-    if (typeof args[args.length - 1] === 'function') {
-      this._callbacks[seq] = args[args.length - 1];
+    const finalArg = args[args.length - 1];
+    if (typeof finalArg === 'function') {
+      this._callbacks[seq] = finalArg;
       args = args.slice(0, -1);
     }
-    this.destFrame.postMessage(
-      {
-        protocol: 'frame-rpc',
-        version: VERSION,
-        sequence: seq,
-        method: method,
-        arguments: args,
-      },
-      this.origin
-    );
+
+    /** @type {RequestMessage} */
+    const message = {
+      arguments: args,
+      method,
+      protocol: PROTOCOL,
+      sequence: seq,
+      version: VERSION,
+    };
+
+    if (this._port) {
+      this._port.postMessage(message);
+    }
+
+    /** @deprecated */
+    if (this.destFrame) {
+      this.destFrame.postMessage(message, this.origin);
+    }
   }
 
   /**
-   * @param {Message} msg
+   * Validate sender
+   *
+   * @param {MessageEvent} event
+   * @deprecated
    */
-  _handle(msg) {
-    if (this._destroyed) {
+  _isValidSender(event) {
```

Comment:
This code predates my changes. What I have done is to extract it to a new method. This means this code was missing test previously.

I can certainly add tests to check that.

On the other hand, this is a function that is has been marked as `@deprecated` so it would go away.

Reply:
Makes sense, then lets skip codecov for this path.